_build/data/transport.core.accesspolicies.php

```diff
@@ -9,136 +9,54 @@
-  'lexicon' => 'permissions',
-], '', true, true);
-
-$policies['11']= $xpdo->newObject(modAccessPolicy::class);
-$policies['11']->fromArray([
-  'id' => 11,
-  'name' => 'Context',
-  'description' => 'A standard Context policy that you can apply when creating Context ACLs for basic read/write and view_unpublished access within a Context.',
-  'parent' => 0,
-  'class' => '',
-  'data' => '{"load":true,"list":true,"view":true,"save":true,"remove":true,"copy":true,"view_unpublished":true}',
-  'lexicon' => 'permissions',
-], '', true, true);
-
-$policies['12']= $xpdo->newObject(modAccessPolicy::class);
-$policies['12']->fromArray([
-    'id' => 12,
-    'name' => 'Hidden Namespace',
-    'description' => 'Hidden Namespace policy, will not show Namespace in lists.',
-    'parent' => 0,
-    'class' => '',
-    'data' => '{"load":false,"list":false,"view":true}',
-    'lexicon' => 'permissions',
-], '', true, true);
+function jsonifyPermissions(array $permissions = []) {
+    $data = [];
+    foreach ($permissions as $key => $permission) {
+        if (is_string($key) && is_bool($permission)) {
+            $data[$key] = $permission;
+        } else {
+            $data[$permission] = true;
+        }
+    }
+    return json_encode($data);
+}
```

Comment:
I agree, that it may be useful, but in this case, need to make this method as static to have the ability to use it in such scripts like this, that looks not good for a class that represents objects.
Move it to a separated helper also not looks good, so I would leave it as is before somebody will really need such functions in another place.

Reply:
Fair enough!